shared/xccdf/services/ssh.xml

```diff
@@ -455,7 +490,28 @@ ossrg="SRG-OS-000163-GPOS-00072,SRG-OS-000279-GPOS-00109" stigid="040340"
 cis="6.2.12" cjis="5.5.6" cui="3.1.11" />
 </Rule>
 
-<Rule id="sshd_disable_rhosts" severity="medium" prodtype="rhel7">
+<Rule id="sshd_set_max_auth_tries" severity="medium" prodtype="rhel7,debian8">
```

Comment:
No need for prodtype here. An empty value will associate this rule (appropriately) with all platforms.